### ISSUE TITLE
RockLib.Messaging.Kafka Dependency [Confluent.Kafka 1.8.2]

### ISSUE THREAD

**Issue by davidpipes** [RockLib/RockLib.Messaging#179]
Is there a reason that we are using an older dependency of Confluent.Kafka (1.8.2). Some of my teams were being advised to use Confluent.Kafka natively instead of RockLib and wanted to get a better understanding if there was any specific reason we are still using 1.8.2.

**Comment by NeilSchiefer** [comment]
Yeah when we did the major .net updates 1.8.2 was the latest.  However, we did recently update dependencies so I'm not sure why that one was missed.  We can likely easily go to 1.9.3, but we'll have to see if there is anything changing in 2+.

**Comment by davidpipes** [comment]
> Yeah when we did the major .net updates 1.8.2 was the latest. However, we did recently update dependencies so I'm not sure why that one was missed. We can likely easily go to 1.9.3, but we'll have to see if there is anything changing in 2+.

Sounds good was just making sure there wasn't a specific compatibility issue that I wasn't aware.of

**Comment by aatchison14** [comment]
https://github.com/RockLib/RockLib.Messaging/issues/179

**Comment by JasonBock** [comment]
Closing as "won't address".
